README.md

```diff
@@ -204,45 +213,45 @@ configure script and makefile (the latter of which just invokes `x.py`).
 make && sudo make install
 ```
 
-When using the configure script, the generated `config.mk` file may override the
-`config.toml` file. To go back to the `config.toml` file, delete the generated
-`config.mk` file.
+When using the configure script, the generated `config.mk` file may override
+the `config.toml` file. To go back to the `config.toml` file, delete the
+generated `config.mk` file.
 
 ## Building Documentation
 
-If you’d like to build the documentation, it’s almost the same:
+If you'd like to build the documentation, it's almost the same:
 
 ```sh
 ./x.py doc
 ```
 
-The generated documentation will appear under `doc` in the `build` directory for
-the ABI used. I.e., if the ABI was `x86_64-pc-windows-msvc`, the directory will be
-`build\x86_64-pc-windows-msvc\doc`.
+The generated documentation will appear under `doc` in the `build` directory
+for the ABI used. I.e., if the ABI was `x86_64-pc-windows-msvc`, the directory
+will be `build\x86_64-pc-windows-msvc\doc`.
 
 ## Notes
 
-Since the Rust compiler is written in Rust, it must be built by a
-precompiled "snapshot" version of itself (made in an earlier stage of
-development). As such, source builds require an Internet connection to
-fetch snapshots, and an OS that can execute the available snapshot binaries.
+Since the Rust compiler is written in Rust, it must be built by a precompiled
+"snapshot" version of itself (made in an earlier stage of development). As
+such, source builds require an Internet connection to fetch snapshots, and an
+OS that can execute the available snapshot binaries.
 
 Snapshot binaries are currently built and tested on several platforms:
 
-| Platform / Architecture                     | x86 | x86_64 |
-|---------------------------------------------|-----|--------|
-| Windows (7, 8, 10, ...)                     | ✓   | ✓      |
-| Linux (kernel 3.2, glibc 2.17 or later)     | ✓   | ✓      |
-| macOS (10.7 Lion or later)                  | (\*) | ✓      |
+| Platform / Architecture                 | x86  | x86_64 |
+|-----------------------------------------|------|--------|
+| Windows (7, 8, 10, ...)                 | ✓    | ✓      |
+| Linux (kernel 3.2, glibc 2.17 or later) | ✓    | ✓      |
+| macOS (10.7 Lion or later)              | (\*) | ✓      |
 
-(\*): Apple dropped support for running 32-bit binaries starting from macOS 10.15 and iOS 11.
-Due to this decision from Apple, the targets are no longer useful to our users.
-Please read [our blog post][macx32] for more info.
+(\*): Apple dropped support for running 32-bit binaries starting from macOS
+10.15 and iOS 11. Due to this decision from Apple, the targets are no longer
+useful to our users. Please read [our blog post][macx32] for more info.
```

Comment:
This is one of the examples of "less good". This is topic-split and now it gets line-broken and the next topic gets connected in unnecessarily.